_implementors/adapters.md

```diff
@@ -42,6 +42,7 @@ The following table lists the known debug adapters that implement the Debug Adap
 [React Native Tools](https://marketplace.visualstudio.com/items?itemName=vsmobile.vscode-react-native)|[@MSLaguana](https://github.com/MSLaguana)|[Microsoft/vscode-react-native](https://github.com/Microsoft/vscode-react-native/issues)
 [Ruby](https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby)|[@rebornix](https://github.com/rebornix)|[rubyide/vscode-ruby](https://github.com/rubyide/vscode-ruby)
 [SWF](https://marketplace.visualstudio.com/items?itemName=bowlerhatllc.vscode-nextgenas)|[@joshtynjala](https://github.com/joshtynjala)|[BowlerHatLLC/vscode-nextgenas](https://github.com/BowlerHatLLC/vscode-nextgenas)
+[NetCoreDbg](https://github.com/Samsung/netcoredbg)|[@ayuckhulk](https://github.com/ayuckhulk)|[Samsung/netcoredbg](https://github.com/Samsung/netcoredbg)

```

Comment:
Should this say the framework/language instead of the the name. So it shoud be
```
[.NET Core](https://github.com/Samsung/netcoredbg)|[@ayuckhulk](https://github.com/ayuckhulk)|[Samsung/netcoredbg](https://github.com/Samsung/netcoredbg)
```

Reply:
I am not sure. But in your example the `.NET Core` link should point to https://github.com/dotnet/coreclr instead of netcoredbg. Maybe just `Debugger for .NET Core`?

Reply:
Yes "Debugger for .NET Core" seems fine as similar pattern is already used in other lines.